infra/build/functions/request_coverage_build.py

```diff
@@ -14,36 +14,38 @@
 #
 ################################################################################
 """Cloud function that requests coverage builds."""
+import base64
 
 import google.auth
 from google.cloud import ndb
 
 import build_and_run_coverage
-from datastore_entities import Project
 import request_build
 
 BASE_PROJECT = 'oss-fuzz-base'
 
 
+def get_build_steps(project_name, image_project, base_images_project):
+  """Retrieve build steps."""
+  project_yaml_contents, dockerfile_lines = request_build.get_project_data(
+      project_name)
+  return build_and_run_coverage.get_build_steps(project_name,
+                                                project_yaml_contents,
+                                                dockerfile_lines, image_project,
+                                                base_images_project)
+
+
 def request_coverage_build(event, context):
   """Entry point for coverage build cloud function."""
-  del event, context  #unused
+  del context  #unused
```

Comment:
Why not just put _ in argument

Reply:
This is recommended per https://google.github.io/styleguide/pyguide.html#214-decision